tasks/list-configured-runners-windows.yml

```diff
@@ -17,8 +17,8 @@
     registered_gitlab_runner_names: "{{ registered_gitlab_runner_names + [json_item['msg']] }}"
   vars:
     json_item: "{{ item | from_json }}"
-  loop: "{{ registered_runners_json_result.stderr.split('\n') }}"
+  loop: "{{ registered_runners_json_result.stderr_lines }}"
```

Comment:
According to this documentation  https://docs.ansible.com/ansible/latest/collections/ansible/windows/win_command_module.html

There is no such field as `stderr_lines`.

Can you confirm that the documentation is wrong and the field is indeed there?

Reply:
I pulled that from here
  https://github.com/riemers/ansible-gitlab-runner/blob/d297222215418ce85a48a468ab87c97db3df25da/tasks/list-configured-runners-unix.yml#L19

And it indeed works, when it was using the split('\n') thing there was a trailing newline, which led to a blank {{ item }} and the task failing.

I think this might be one of those ansible "it works everywhere" kinda things.

Reply:
![image](https://github.com/riemers/ansible-gitlab-runner/assets/1463882/688f3c1b-c6b5-49b4-a872-8275cdf0644c)

Reply:
I'm not 1000% sure, but this implies you can expect `stderr_lines` to always be present.
https://docs.ansible.com/ansible/latest/reference_appendices/common_return_values.html#stderr-lines 

Probably just a case of the ansible community windows missing some docs on things they consider to be "expected"

Reply:
Upstream PR :)
https://github.com/ansible-collections/ansible.windows/pull/550

Reply:
Awesome.